package.json

```diff
@@ -8,15 +8,29 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-relay": "^10.1.1",
+    "relay": "^0.8.0-1"
+  },
+  "scripts": {
+    "relay": "relay-compiler --src ./src --schema schema.graphql --extensions ts tsx --language typescript",
+    "start": "parcel src/index.html",
```

Comment:
This is why I really love prototyping things out w/ parcel. The build setup is just _so simple_.